geniesp/bpc_redcap_export_mapping.py

```diff
@@ -433,6 +437,33 @@ def download_metadata_files(self):
                     ifcollision="overwrite.local",
                 )
 
+    def create_bpc_cbio_metafiles(self):
+        """Create BPC cBioPortal meta* files"""
+        mg_release_ent = self.syn.get(self._MG_RELEASE_SYNID)
+        name = f"GENIE BPC {self._SPONSORED_PROJECT} v{self.release}"
+        description = (
+            f"{self._SPONSORED_PROJECT} cohort v{self._SPONSORED_PROJECT} "
+            f"(GENIE {date.today().year}) GENIE {mg_release_ent.name}"
+        )
+        short_name = f"{self._SPONSORED_PROJECT} GENIE"
+        study_identifier = f"{self._SPONSORED_PROJECT.lower()}_genie_bpc"
+        metafiles.create_cbio_metafiles(
+            study_identifier=study_identifier, outdir=self._SPONSORED_PROJECT
```

Comment:
TODO: Still need to figure out how to pass in the list of cbioPortal files generated.  I know we had discussed adding to the configuration of the class a list of cBioPortal files required for each cohort, but currently the only way we would really utilize this list is:
```
if filename in list_of_files_to_generate:
     write_file
```